<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LivreDetail</class>
 <widget class="QMainWindow" name="LivreDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>621</width>
      <height>421</height>
     </rect>
    </property>
    <widget class="QWidget" name="nouveauPage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Titre : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="titreEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Auteur : </string>
      </property>
     </widget>
     <widget class="QComboBox" name="auteurComboBox">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Publication du livre  : </string>
      </property>
     </widget>
     <widget class="QDateEdit" name="publicationDateEdit">
      <property name="geometry">
       <rect>
        <x>229</x>
        <y>210</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>150</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nombre de page  : </string>
      </property>
     </widget>
     <widget class="QSpinBox" name="pageSpinBox">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>150</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="editeurComboBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Editeur : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Genre : </string>
      </property>
     </widget>
     <widget class="QComboBox" name="genreComboBox">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>80</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Langue : </string>
      </property>
     </widget>
     <widget class="QComboBox" name="langueComboBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>210</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Quantit√©  : </string>
      </property>
     </widget>
     <widget class="QSpinBox" name="quantiteSpinBox">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>210</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Synopsis : </string>
      </property>
     </widget>
     <widget class="QTextEdit" name="resumeTextEdit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>270</y>
        <width>501</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>380</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1</y>
        <width>621</width>
        <height>431</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/res/img/background.png);</string>
      </property>
     </widget>
     <zorder>widget</zorder>
     <zorder>label</zorder>
     <zorder>titreEdit</zorder>
     <zorder>label_2</zorder>
     <zorder>auteurComboBox</zorder>
     <zorder>label_3</zorder>
     <zorder>publicationDateEdit</zorder>
     <zorder>label_4</zorder>
     <zorder>pageSpinBox</zorder>
     <zorder>editeurComboBox</zorder>
     <zorder>label_5</zorder>
     <zorder>label_6</zorder>
     <zorder>genreComboBox</zorder>
     <zorder>label_7</zorder>
     <zorder>langueComboBox</zorder>
     <zorder>label_8</zorder>
     <zorder>quantiteSpinBox</zorder>
     <zorder>label_9</zorder>
     <zorder>resumeTextEdit</zorder>
     <zorder>pushButton</zorder>
    </widget>
    <widget class="QWidget" name="detailPage"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
